From 86d300ff94bd30d508f768f808b3b083cfb39cef Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Thu, 16 Jul 2020 13:06:24 +0700
Subject: [PATCH 085/504] ARM: dts: iwg20d-common: Modify pfc of sdhi1

This patch adds sdhi1_cd to pin groups
and move sdhi_pins_uhs to right position.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm/boot/dts/iwg20d-q7-common.dtsi | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/iwg20d-q7-common.dtsi b/arch/arm/boot/dts/iwg20d-q7-common.dtsi
index 48f8fa1..b370e38 100644
--- a/arch/arm/boot/dts/iwg20d-q7-common.dtsi
+++ b/arch/arm/boot/dts/iwg20d-q7-common.dtsi
@@ -194,22 +194,22 @@
 	};
 
 	sdhi1_pins: sd1 {
-		groups = "sdhi1_data4", "sdhi1_ctrl";
+		groups = "sdhi1_data4", "sdhi1_ctrl", "sdhi1_cd";
 		function = "sdhi1";
 		power-source = <3300>;
 	};
 
+	sdhi1_pins_uhs: sd2_uhs {
+		groups = "sdhi1_data4", "sdhi1_ctrl", "sdhi1_cd";
+		function = "sdhi1";
+		power-source = <1800>;
+	};
+
 	sound_pins: sound {
 		groups = "ssi0129_ctrl", "ssi0_data", "ssi1_data";
 		function = "ssi";
 	};
 
-	sdhi1_pins_uhs: sd2_uhs {
-		renesas,groups = "sdhi1_data4", "sdhi1_ctrl";
-		renesas,function = "sdhi1";
-		power-source = <1800>;
-	};
-
 	usb0_pins: usb0 {
 		groups = "usb0";
 		function = "usb0";
-- 
2.7.4

